### PR TITLE
[BUGFIX beta] Fix Error object's stacktrace inconsistency across browsers

### LINKED ISSUE #12876: Thrown Errors in Firefox doesn't include the error message
I posted this question while trying to figure it out myself:
http://stackoverflow.com/q/35071984/359532

From what I can tell, in chrome the 'stack' property contains the error message while Firefox does not. 

In my case, the error handler is from `ember-runtime/ext/rsvp`. I think to fix this, `onerrorDefault()` could print the error message and the stack trace, instead of just the stack trace. However, in chrome it would appear twice.

### PR DESCRIPTION
**Pull request by Charizard**
Fixes #12876

### CODE REVIEW

**Comment by rwjblue** [comment]
Seems good to me, @stefanpenner r?

File: packages/ember-runtime/lib/ext/rsvp.js [hunk 1]
```diff
@@ -62,14 +70,14 @@ export function onerrorDefault(reason) {
 
       if (Test && Test.adapter) {
         Test.adapter.exception(error);
-        Logger.error(error.stack);
+        Logger.error(getStack(error));
```
Comment (ryanpatrickcook) [inline packages/ember-runtime/lib/ext/rsvp.js]: Could this be `Logger.error(error)`?

**Comment by stefanpenner** [comment]
reason reasonable, but if stack is null/undefined should we bothing logging it?

**Comment by Charizard** [comment]
@stefanpenner Are there cases, where stack can be `null`/`undefined`?

**Comment by Turbo87** [comment]
FWIW I think checking `error.stack.indexOf(error.message) === -1` will be more robust

**Comment by rwjblue** [comment]
Agreed

**Comment by Charizard** [comment]
Thanks to @Turbo87, I have updated the PR.

**Comment by rwjblue** [comment]
Thanks @Charizard!

**Comment by Charizard** [comment]
Thanks to you @rwjblue and the whole ember team for such an awesome framework 😄

**Pull request merged**